ideas/322-dao.md

```diff
@@ -0,0 +1,76 @@
+---
+id: #322-Dao
+title: DAO
+status: research
+lead-contributor: Barry @bgits
+contributors:
+    - Barry @bgits
+    - Iuri @iurimatias
+    - Jarrad @jarradhope
+    - Ricardo @3esmit
+    - Richard @richard-ramos
+budget:
+- actual: xxx
+- estimate: yyy
+- currency: ETH/USD/SNT
+---
+
+
+DAO
+=
+
+## Summary and Goals
+
+One major drawback in legacy social networks is the lack of influence their users possess over the networks themselves. They are often powerless in having a say on how the platform evolves. We aim to democratise this power, giving stakeholders a direct influence over all decisions within the network, including how the software is developed with voting and allocation of funds via liquid pledging
+
+
+## Contributors
+
+- Barry @bgits
+- Iuri @iurimatias
+- Jarrad @jarradhope
+- Ricardo @3esmit
+- Richard @richard-ramos
+
+(+ Janitor swarm)
+
+## Communication
+
+- `status channel`: #status-dao
+- `sync schedule`: brainstorming on Mondays / stand ups on Wednesday
+- `pivotal link`: https://www.pivotaltracker.com/n/projects/2231548
+- `research notes`: #status-dao
+- `github repo`: 
+- - https://github.com/status-im/liquidpledging/tree/embark-dapp
+- - https://github.com/status-im/snt-voting
+
+## Research
```

Comment:
I would add:

- What information is important to be displayed to funders, delegates and projects?
- What is the ideal specification for a "Pledge"?